commit 151da5a562b9ddbe57dcbd24cb3d855faf843516
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Tue Oct 5 09:56:50 2010 -0700

    Staging: brcm80211: remove unit64 and int64 typedefs
    
    They are not used anywhere, nor should they be defined.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/typedefs.h
+++ b/drivers/staging/brcm80211/include/typedefs.h
@@ -70,10 +70,6 @@ typedef unsigned short uint16;
 typedef unsigned int uint32;
 #endif
 
-#ifndef TYPEDEF_UINT64
-typedef unsigned long long uint64;
-#endif
-
 #ifndef TYPEDEF_UINTPTR
 typedef unsigned int uintptr;
 #endif
@@ -90,11 +86,6 @@ typedef signed short int16;
 typedef signed int int32;
 #endif
 
-#ifndef TYPEDEF_INT64
-typedef signed long long int64;
-#endif
-
-
 /* define macro values */
 
 #ifndef FALSE
@@ -126,12 +117,10 @@ typedef signed long long int64;
 #undef TYPEDEF_UINT8
 #undef TYPEDEF_UINT16
 #undef TYPEDEF_UINT32
-#undef TYPEDEF_UINT64
 #undef TYPEDEF_UINTPTR
 #undef TYPEDEF_INT8
 #undef TYPEDEF_INT16
 #undef TYPEDEF_INT32
-#undef TYPEDEF_INT64
 #undef TYPEDEF_FLOAT32
 #undef TYPEDEF_FLOAT64
 #undef TYPEDEF_FLOAT_T
